media-domain/src/main/java/nl/vpro/domain/media/MediaObjects.java

```diff
@@ -941,9 +929,108 @@ public static boolean revokeRelatedPublishables(MediaObject media, Instant now)
                 .flatMap(Collection::stream)
                 .map(Topic::getGtaaRecord)
         ).distinct();
+    }
+
+    /**
+     * Whether this mediaobject is playable in a NPO player.
+     * @since 5.11
+     */
+    public static boolean isPlayable(MediaObject media) {
+        if (media == null) {
+            return false;
+        }
+        // TODO, it seems that this is enough (at least for video?)
```

Comment:
Willen we deze nog oplossen? Zo niet dan kunnen we hier een ticket voor maken

Reply:
Dit is geen nieuwe code.  Deze method bleek er al te staan. Heb hem alleen verplaatst zodat hij naast de nieuwe staat, want het is sowieso iets dergelijks.